libs/src/services/creatorNode/index.js

```diff
@@ -737,6 +737,13 @@ class CreatorNode {
 
     const requestId = uuid()
     headers['X-Request-ID'] = requestId
+
+    const user = this.userStateManager.getCurrentUser()
+    if (user && user.wallet && user.user_id) {
+      headers['User-Wallet-Addr'] = user.wallet
```

Comment:
ah i didn't know that, and yeah doesn't make sense to address now but i added a TODO and a linear card
since both are already consumed in CN maybe the right rollout would be to add duplicate headers with the X- prefix and transition after some time